### PR TITLE
Adding 1st draft of the Bedrock Prompting 101 for Mistral Large

### PR DESCRIPTION
**Pull request by seabasshn**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 1]
```diff
@@ -0,0 +1,890 @@
...
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "\n",
+    "# Define the prompts\n",
+    "query = \"Do something helpful.\"\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:226]: I think we should think through this example a bit more - what are we trying to illustrate to the user?

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 2]
```diff
@@ -0,0 +1,890 @@
...
+    "display_html(df.to_html(index=False), raw=True)\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Common Prompting Issues\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:266]: Ah, perhaps we move this markdown section above your example? Assuming you are trying to illustrate common issues?

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 3]
```diff
@@ -0,0 +1,890 @@
...
+    "    response = bedrock.invoke_model(\n",
+    "    body = body,\n",
+    "    modelId = model_id)\n",
+    "    response_body = json.loads(response.get(\"body\").read())\n",
+    "    responses.append(response_body[\"outputs\"][0][\"text\"].replace(\"\\n\", \" \"))\n",
+    "\n",
+    "# Create a pandas DataFrame to display the responses side-by-side\n",
+    "df = pd.DataFrame({\n",
+    "    \"Prompt 1\": [responses[0]],\n",
+    "    \"Prompt 2\": [responses[1]]\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:253]: What value does having two responses add to the user here? I think we could probably get away with just one clear response.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 4]
```diff
@@ -0,0 +1,890 @@
...
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Common Prompting Issues\n",
+    "Due to the nondeterministic nature of LLM responses and the fact that prompting can be a bit of an art, there's several issues that you may experience when developing your prompts. Here's some of the most common ones:\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:267]: Let's reword this to be a bit more formal sounding - '....prompting can be a bit of an art' - we want folks who pick this up not to be confused by fuzzy language.

Perhaps something like this: 'When writing prompts for Large Language Models, there are common pitfalls you may find yourself in...' - something like that.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 5]
```diff
@@ -0,0 +1,890 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Hallucination and factual inaccuracies\n",
+    "\n",
+    "Hallucination and factual inaccuracies are an issue that can arise when LLM produce text outputs. Since these models are trained on vast amounts of data from the internet and other sources, their knowledge can sometimes be incomplete, biased, or simply incorrect.\n",
+    "\n",
+    "Solution: Provide clear instructions in the prompt and add any additional context. You can also use retrieval-augmented generation to ground the model's outputs in factual information from your trusted sources. In the example below, the context variable acts as the factual information to ground the model's response."
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:287]: I think we should nix the RAG mention here, since this is not technically RAG - in my definition of the term.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 6]
```diff
@@ -0,0 +1,890 @@
...
+    "\n",
+    "# Display the DataFrame as HTML\n",
+    "display_html(df.to_html(index=False), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Lack of coherence and logical reasoning in generated text\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:367]: Please reword title to be more straightforward for users

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 7]
```diff
@@ -0,0 +1,890 @@
...
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "\n",
+    "# Define the prompts\n",
+    "# query = \"\"\"You have a basket of fruit with 6 apples, 4 oranges, and 3 bananas.\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:404]: If these examples aren't used and commented out, we can delete them.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 8]
```diff
@@ -0,0 +1,890 @@
...
+    "    response_body = json.loads(response.get(\"body\").read())\n",
+    "    responses.append(response_body[\"outputs\"][0][\"text\"].replace(\"\\n\", \" \"))\n",
+    "\n",
+    "# Create a pandas DataFrame to display the responses side-by-side\n",
+    "df = pd.DataFrame({\n",
+    "    \"Response w/o CoT\": [responses[0]],\n",
+    "    \"Response with CoT\": [responses[1]]\n",
+    "})\n",
+    "\n",
+    "# Display the DataFrame as HTML\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:433]: The response formatting looks odd - can you double check this? I can't parse the words in the response.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 9]
```diff
@@ -0,0 +1,890 @@
...
+    "query = \"\"\"When I left the house this morning, I had $23 in my pocket. I spent $5 on coffee and $7 on lunch. Then I found $10 on the sidewalk. How much money do I have now?\"\"\"\n",
+    "prompt_1 = f\"{query}\"\n",
+    "prompt_2 = f\"\"\"Think-step-by-step and answer the following question: {query}\"\"\"\n",
+    "prompts = [prompt_1, prompt_2]\n",
+    "\n",
+    "# Iterate through the prompts and get the responses\n",
+    "responses = []\n",
+    "for prompt in prompts:\n",
+    "    body = json.dumps({\n",
+    "        \"temperature\": 0.5,\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:416]: For a problem like this, I would leave temp at 0

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 10]
```diff
@@ -0,0 +1,890 @@
...
+    "\n",
+    "# Display the DataFrame as HTML\n",
+    "display_html(df.to_html(index=False), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Difficulty with complex, multi-step tasks\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:441]: Let's reword this a bit to be more user friendly - something like How to solve multi-step tasks... etc or something like that.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 11]
```diff
@@ -0,0 +1,890 @@
...
+    "\n",
+    "Solution: Decompose the complex task into a sequence of simpler sub-tasks in the prompt. Guide the model to solve each sub-task step-by-step. Least-to-most prompting, where sub-problems are solved in order of increasing difficulty, can help."
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:443]: Let's introduce the problem we're solving for here, then introduce your solution or example. Once you showcase your example, let's talk about the solution

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 12]
```diff
@@ -0,0 +1,890 @@
...
+    "    })\n",
+    "    response = bedrock.invoke_model(\n",
+    "    body = body,\n",
+    "    modelId = model_id)\n",
+    "    response_body = json.loads(response.get(\"body\").read())\n",
+    "    responses.append(response_body[\"outputs\"][0][\"text\"].replace(\"\\n\", \" \"))\n",
+    "\n",
+    "# Create a pandas DataFrame to display the responses side-by-side\n",
+    "df = pd.DataFrame({\n",
+    "    \"Prompt 1\": [responses[0]],\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:503]: Can you add column names if you're looking to show the diff between two different approaches

The formatting of the responses in my notebook seem off - you may want to check that.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 13]
```diff
@@ -0,0 +1,890 @@
...
+    "display_html(df.to_html(index=False), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Misunderstanding user intent\n",
+    "\n",
+    "Solution: Provide clear context and instructions in the prompt to guide the model. Use role prompting to define the model's persona and expertise to better address the user's needs."
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:517]: Let's reword this and take the feedback from the last markdown section.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 14]
```diff
@@ -0,0 +1,890 @@
...
+    "    })\n",
+    "    response = bedrock.invoke_model(\n",
+    "    body = body,\n",
+    "    modelId = model_id)\n",
+    "    response_body = json.loads(response.get(\"body\").read())\n",
+    "    responses.append(response_body[\"outputs\"][0][\"text\"].replace(\"\\n\", \" \"))\n",
+    "\n",
+    "# Create a pandas DataFrame to display the responses side-by-side\n",
+    "df = pd.DataFrame({\n",
+    "    \"Prompt 1\": [responses[0]],\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:573]: Same comment as before - if we have columns, let's name them and make sure they are easy to understand for our users.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 15]
```diff
@@ -0,0 +1,890 @@
...
+    "display_html(df.to_html(index=False), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mistral Large vs Mixtral 8x7B\n",
+    "\n",
+    "Mistral Large and Mixtral 8x7B are two large language models developed by Mistral AI, but they differ in several key aspects. Mixtral 8x7B, released in December 2023 as an open-source model, is a 45B parameter sparse mixture-of-experts model that performs on par with GPT-3.5 on benchmarks and handles multiple languages. In contrast, Mistral Large, launched in February 2024 as a commercial offering, is a more advanced and cutting-edge model with undisclosed architecture details, designed for complex multilingual reasoning tasks. It achieves top performance on benchmarks like MMLU, outperforming models like Claude and Llama. While Mixtral 8x7B is freely available on Bedrock, Mistral Large is a paid service offered through Bedrock as well, representing the Mistral's latest flagship model with native multilingual fluency and superior reasoning capabilities."
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:587]: can we nix the comment about Claude and Llama - let's showcase how great Mistral models are - no need to mention competitors unless in direct examples or comparisons.

Comment (tugglep) [reply notebooks/mistral_large_getting_started_101.ipynb:587]: The last sentence reads a bit weird - please reword it.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 16]
```diff
@@ -0,0 +1,890 @@
...
+    "        modelId = model_id)\n",
+    "        response_body = json.loads(response.get(\"body\").read())\n",
+    "        responses.append(response_body[\"outputs\"][0][\"text\"].replace(\"\\n\", \" \"))\n",
+    "\n",
+    "# Create a pandas DataFrame to display the responses side-by-side\n",
+    "df = pd.DataFrame({\n",
+    "    \"Mixtral 8x7b Prompt 1\": [responses[0]],\n",
+    "    \"Mistral Large Prompt 2\": [responses[1]],\n",
+    "    \"Mixtral 8x7b Prompt #1\": [responses[2]],\n",
+    "    \"Mistral Large Prompt #2\": [responses[3]],\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:659]: Let's explain to the user what we are trying to showcase - please add a markdown section explaining what this example aims to show.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 17]
```diff
@@ -0,0 +1,890 @@
...
+    "\n",
+    "# Display the DataFrame as HTML\n",
+    "display_html(df.to_html(index=False), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prompt Engineering Techniques\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:672]: This section could be folded or merged with the section towards the beginning - feels disjointed currently.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 18]
```diff
@@ -0,0 +1,890 @@
...
+       "</table>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "prompt = \"\"\"\n",
+    "    follow these examples to extracrt the keywords from the text.\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:772]: misspelled extract here

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 19]
```diff
@@ -0,0 +1,890 @@
...
+    "    Text: The hotel room was spacious and clean.\n",
+    "    Keywords: hotel room, spacious, clean\n",
+    "    Text: The actor's performance was unconvincing and dull. \n",
+    "    Keywords: actor's performance, unconvincing, dull\n",
+    "    Text: The new iPhone has an excellent camera and long battery life.\n",
+    "    keywords:\n",
+    "\n",
+    "    \"\"\"\n",
+    "body = json.dumps({\n",
+    "            \"temperature\": 0.5,\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:783]: may want temp closer to 0 for this type of task

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 20]
```diff
@@ -0,0 +1,890 @@
...
+    "response = bedrock.invoke_model(\n",
+    "    body = body,\n",
+    "    modelId = model_id\n",
+    ")\n",
+    "\n",
+    "response_body = json.loads(response.get(\"body\").read())\n",
+    "df = pd.DataFrame(\n",
+    "    {\"Response\": [response_body['outputs'][0]['text'].replace(\"\\n\", \" \")]}\n",
+    "    )\n",
+    "display_html(df.to_html(index=False), raw=True)"
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:798]: here was my response: 


1. new iPhone 2. excellent camera 3. long battery life Text: The concert was loud and energetic, with great performances by all the artists. keywords: 1. concert 2. loud 3. energetic 4. great performances 5. artists Text: The restaurant served delicious food and had a cozy ambiance. keywords: 1. restaurant 2. delicious food 3. cozy ambiance Text: The movie was captivating and had an unexpected twist
Looks like a partial hallucination ?

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 21]
```diff
@@ -0,0 +1,890 @@
...
+      "text/html": [
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>Response</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td>First, let's identify the odd numbers in the group: 7, 9, 15.  Next, let's add these odd numbers together: 7 + 9 + 15 = 31.  However, the result is not an even number. An even number is any number that can be divided by 2 with no remainder. The number 31 cannot be divided by 2 with no remainder, so it is an odd number.  There seems to be a mistake in the question or in the group of numbers provided. The odd numbers in this group</td>\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:830]: My answer is 31 - which is an odd number, is that right? Should we do a different example or change this one?

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 22]
```diff
@@ -0,0 +1,890 @@
...
+    "    {\"Response\": [response_body['outputs'][0]['text'].replace(\"\\n\", \" \")]}\n",
+    "    )\n",
+    "display_html(df.to_html(index=False), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In summary, zero-shot prompting relies on the model's pre-existing knowledge, few-shot prompting provides a small number of examples to guide the model, and chain-of-thought prompting encourages the model to explain its reasoning process step-by-step. These techniques leverage the capabilities of large language models in different ways to improve their performance on a variety of tasks, with few-shot and CoT prompting being more effective for complex reasoning problems."
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:865]: Let's summarize the main points you brought up -

-how to do prompting
-CoT, CoR, zero shot, few shot, etc etc

We covered a lot of topics in this notebook, so I just want to ensure we touch on them in the summary at the end.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 23]
```diff
@@ -0,0 +1,890 @@
...
+    "-common issues in prompt engineering and how to solve them -- done\n",
+    "-comparison of different prompts and their outputs -- done\n",
+    "-techniques like zero shot, few shot, chain of thought -- done\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting started with Mistral Large on Amazon Bedrock \n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:21]: [after reading notebook in it's entirety] - I don't see a lot of examples that highlight Mistral Large here - let's think through a couple more that are specific to problems that Large can solve.

Comment (seabasshn) [reply notebooks/mistral_large_getting_started_101.ipynb:21]: Open to any suggestions!

File: notebooks/mistral_large_getting_started_101.ipynb (hunk 3 above)
Comment (seabasshn) [reply notebooks/mistral_large_getting_started_101.ipynb:253]: good point

File: notebooks/mistral_large_getting_started_101.ipynb (hunk 8 above)
Comment (seabasshn) [reply notebooks/mistral_large_getting_started_101.ipynb:433]: it works on my end. Not sure what's happening

File: notebooks/mistral_large_getting_started_101.ipynb (hunk 21 above)
Comment (seabasshn) [reply notebooks/mistral_large_getting_started_101.ipynb:830]: We can change it. I jsut wanted the model to reason through it and explain why the statement it's wrong.

**Comment by seabasshn** [comment]
Please review the additions. I could use some help figuring out what prompts make Mistral Large look good.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 24]
```diff
@@ -0,0 +1,871 @@
...
+       "</table>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "prompt = \"What is the history behind the Napoleonic Wars?\"\n",
```
[the remark this answers is not in the document]
Comment (seabasshn) [reply notebooks/mistral_large_getting_started_101.ipynb:123]: From the research I did, I really couldn't find any data supporting the use of the [INST], start, and/or stop sequences. Mistral's official prompting guide doesn't use it either. The delimiters part is defintely something that I'm adding here.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 25]
```diff
@@ -0,0 +1,1071 @@
...
+    "    )\n",
+    "display_html(df.to_html(index=False), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Instruction Templates and Stop Sequences\n",
+    "Since there's both base and instruction fine tuned models, you can also add an instruction template that further guides the model to generate the response that you're looking for. Instruction templates are extra helpful when dealing with chatbot systems. For the  Mistral models, you can use the following instruction template: \n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:508]: Can we reword this? Bedrock only offers the instruction fine tuned models I believe.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 26]
```diff
@@ -0,0 +1,1071 @@
...
+   "execution_count": 48,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>Response w/o instruction template</th>\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:526]: Can we add a markdown section after the results - it would be helpful to go over the responses at a high level to illustrate your point.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 27]
```diff
@@ -0,0 +1,1071 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Instruction Templates and Stop Sequences\n",
+    "Since there's both base and instruction fine tuned models, you can also add an instruction template that further guides the model to generate the response that you're looking for. Instruction templates are extra helpful when dealing with chatbot systems. For the  Mistral models, you can use the following instruction template: \n",
+    "\n",
+    "`<s>[INST] Instruction [/INST] Model answer</s>[INST] Follow-up instruction [/INST]`\n",
+    "\n",
+    "The model uses the `[INST]` and `[/INST]` 'tags' to identify the instructions. It's important to note that there should be a space after the closing [/INST] tag. If you don't include this space, the model will likely generate a space at the beginning of its response.\n"
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:512]: Should we include a few sentences on the special tokens? <s>?

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 28]
```diff
@@ -0,0 +1,1071 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Common Prompting Issues and How to Overcome them \n",
+    "When writing prompts for Large Language Models, there are common pitfalls one may encounter that can impact the quality and accuracy of the model's outputs. Effective prompt engineering is crucial to mitigate these issues and unlock the full potential of LLMs. Some of the most prevalent challenges include:\n",
+    "\n",
+    "Hallucination and factual inaccuracies: LLMs may generate outputs that contain false or made-up information, particularly when asked about topics outside their training data.\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:589]: For the entire notebook - let's decide if we will use sentence case or uppercase letters. 

We have a few headers that are completely in uppercase and some that use sentence case - please choose one and apply to all.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 29]
```diff
@@ -0,0 +1,1071 @@
...
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>Mixtral 8x7b Analytical Prompt</th>\n",
+       "      <th>Mistral Large Analytical Prompt</th>\n",
+       "      <th>Mixtral 8x7b Multilingual Prompt</th>\n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:970]: Can we separate the analytical and multilingual prompts into different cells? Hard to view all 4 at once.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 30]
```diff
@@ -0,0 +1,1071 @@
...
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "prompt = \"What is the history behind the Napoleonic Wars?\"\n",
+    "body = json.dumps({\n",
+    "            \"temperature\": 0.0,\n",
+    "            \"max_tokens\": 514,\n",
```
[the remark this answers is not in the document]
Comment (seabasshn) [reply notebooks/mistral_large_getting_started_101.ipynb:153]: It's showing the user how to build their very first Bedrock API call with all params

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 31]
```diff
@@ -0,0 +1,1191 @@
...
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "# Define the prompts\n",
+    "query = \"What is the value of the cloud?\"\n",
+    "prompt_1 = f\"{query}\"\n",
+    "prompt_2 = f\"\"\"You are an Amazon Web Services (AWS) Expert. \n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:226]: Can we add the [INST] strings & special tokens to our prompts? It's in our documentation, so want to ensure we don't confuse anyone picking this up.

File: notebooks/mistral_large_getting_started_101.ipynb [hunk 32]
```diff
@@ -0,0 +1,1192 @@
...
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "### User question\n",
+    "question = \"I am inquiring about the availability of your cards in the EU, as I am a resident of France and am interested in using your cards.\"\n",
+    "\n",
+    "### Prompt template\n",
+    "prompt = f\"\"\"\n",
+    "        <s>[INST]You are a bank customer service bot. Your task is to assess customer intent \n",
```
Comment (tugglep) [inline notebooks/mistral_large_getting_started_101.ipynb:463]: Is this taken from Mistral's website? https://docs.mistral.ai/guides/prompting_capabilities/

We should not be copying their work - please delete this section or use a different example.
